EmbedVideo.hooks.php

```diff
@@ -138,6 +140,7 @@ public function onParserFirstCallInit($parser) {
 		// smart handling of service name tags (if they aren't already implamented)
 		$tags = $parser->getTags();
 		$services = \EmbedVideo\VideoService::getAvailableServices();
+		$wgEmbedVideoEnabledServices = $services;
```

Comment:
I don't understand the purpose of the `$wgEmbedVideoEnabledServices` variable. It is always overridden here? 🤔